__tests__/app.test.js

```diff
@@ -247,13 +247,22 @@ describe("PATCH /api/articles/:article_id", () => {
         expect(article.votes).toBe(90);
       });
   });
+  test("400: Responds with an error message if incorrect data type is used for votes", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: "vote" })
+      .expect(400)
+      .then(({ body: { message } }) => {
+        expect(message).toBe("Bad request - incorrect data type");
```

Comment:
This would be fine to just be bad request, we don't want to be too explicit on error messages.